itests/kit/blockminer.go

```diff
@@ -245,7 +246,9 @@ func (bm *BlockMiner) MineBlocksMustPost(ctx context.Context, blocktime time.Dur
 				dlinfo, err := bm.miner.FullNode.StateMinerProvingDeadline(ctx, minerAddr, ts.Key())
 				require.NoError(bm.t, err)
 				require.NotNil(bm.t, dlinfo, "no deadline info for miner %s", minerAddr)
-				impendingDeadlines = append(impendingDeadlines, minerDeadline{addr: minerAddr, deadline: *dlinfo})
+				if dlinfo.Open <= dlinfo.CurrentEpoch {
```

Comment:
This should be `dlinfo.Open` < `dlinfo.CurrentEpoch` right ? It is fine if it's == `dlinfo.CurrentEpoch`